feat(Grid): Allow components to adapt their appearance to the width of their Grid Cell (#2543)
Amsterdam/design-system: packages/css/src/components/grid/README.md

- Grids must not be nested.
  The component is currently intended for the full width of the page only.
- The Gap utility classes must not be used on the Grid component.

> - Components can base their appearance on the width of a ancestor Grid Cell using [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Guides/Containment/Container_queries).

Reviewer: Can we simplify a bit more?

```suggestion
- Components can adapt their appearance to the width of their Grid Cell through a [CSS container query](https://developer.mozilla.org/en-US/docs/Web/CSS/Guides/Containment/Container_queries).
```

Author: Done!

> - Components can base their appearance on the width of an ancestor Grid Cell using [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Guides/Containment/Container_queries).

Author: ```suggestion
- Components can adapt their appearance to the width of their Grid Cell through a [CSS container query](https://developer.mozilla.org/en-US/docs/Web/CSS/Guides/Containment/Container_queries).
```

Re-apply suggestion by @VincentSmedinga after conflicting committed suggestion from Copilot.
